server/src/main/resources/db/changelog/20191218180815-create-provided-product-mappings.xml

```diff
@@ -48,43 +48,20 @@
     </changeSet>
 
     <changeSet id="20191218180815-2" author="sdhome">
-        <comment>Create table for derived product and derived provided products mappings</comment>
-
-        <createTable tableName="cp2_product_derprov_products">
-            <column name="product_uuid" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-
-            <column name="provided_product_uuid" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-
-        </createTable>
-
-        <addForeignKeyConstraint
-            baseTableName="cp2_product_derprov_products"
-            baseColumnNames="product_uuid"
-            constraintName="cp2_prod_derprov_products_fk1"
-            deferrable="false"
-            initiallyDeferred="false"
-            onDelete="CASCADE"
-            onUpdate="NO ACTION"
-            referencedColumnNames="uuid"
-            referencedTableName="cp2_products"
-            referencesUniqueColumn="false" />
-
-        <addForeignKeyConstraint
-            baseTableName="cp2_product_derprov_products"
-            baseColumnNames="provided_product_uuid"
-            constraintName="cp2_prod_derprov_products_fk2"
-            deferrable="false"
-            initiallyDeferred="false"
-            onDelete="CASCADE"
-            onUpdate="NO ACTION"
-            referencedColumnNames="uuid"
-            referencedTableName="cp2_products"
-            referencesUniqueColumn="false" />
-
+        <comment>Migrate data of cp2_pool_provided_products and cp2_pool_derprov_products
+            to cp2_product_provided_products table</comment>
+        <sql>
+            INSERT into cp2_product_provided_products
+                SELECT      pool.product_uuid,mapping.product_uuid provided_product_id
```

Comment:
If we're going to format this as nicely as it is, let's also add the optional alias indicators for provided_product_id

Reply:
Done. added the alias indicator.